Cron/Images.php

```diff
@@ -122,12 +90,8 @@ public function execute()
                 // validate returned images:
                 if(is_array($images->{'Items'}) && count($images->{'Items'}) > 0) {
                     // download and save the images locally:
-                    $this->saveImageForProduct($images->{'Items'});
-                    // update the processed images configuration:
-                    $processedEans = array_merge($processedEans, $eans);
-                    $encodedEans = json_encode($processedEans);
-                    $this->saveImagesEans($encodedEans);
-                    $this->logger->info('New images synchronized.');
+                    $newImagesCount = $this->imagesHelper->saveProductImages($images->{'Items'});
+                    $this->logger->info('New synchronized images: '.$newImagesCount);
                 }
             }
         } catch (Exception $e) {
```

Comment:
I think it's better to throw the error out of the execute method. Magento will catch it and log it properly. Also it would be easier to debug.